src/Air.zig

```diff
@@ -264,6 +264,13 @@ pub const Inst = struct {
         /// Given a pointer to a struct and a field index, returns a pointer to the field.
         /// Uses the `ty_pl` field, payload is `StructField`.
         struct_field_ptr,
+        /// Same as above, but for common cases to save extra data memory.
```

Comment:
Generally doc comments should reference stuff by name rather than saying "above" or "below" in case someone is looking at documentation out of context, or e.g. looking at generated docs of only 1 field.

Likewise they should be duplicated even for similar things, such as each of these new tags.